plugins/reach/lib/model/VendorCatalogItem.php

```diff
@@ -194,8 +194,12 @@ public function calculatePriceForEntry(entry $entry, $taskDuration = null)
 		return call_user_func($this->getPricing()->getPriceFunction(), $durationMsec, $this->getPricing()->getPricePerUnit());
 	}
 	
-	public function getTaskVersion($entryId, $jobData = null)
+	public function getTaskVersion($entryId, $entryObjectType = KalturaEntryObjectType::ENTRY, $jobData = null)
 	{
+		if($entryObjectType != KalturaEntryObjectType::ENTRY)
```

Comment:
Lets move the old code to be getEntryTaskVersion, and the new function will decide which function to call base on the type, if its entry call getEntryTaskVersion(), otherwise do nothing (for now)